deepface/basemodels/OpenFace.py

```diff
@@ -1,250 +1,535 @@
-	
-	home = str(Path.home())
-	
-	if os.path.isfile(home+'/.deepface/weights/openface_weights.h5') != True:
-		print("openface_weights.h5 will be downloaded...")
-		
-		url = 'https://drive.google.com/uc?id=1LSe1YCV1x-BfNnfb7DFZTNpv_Q9jITxn'
-		output = home+'/.deepface/weights/openface_weights.h5'
-		gdown.download(url, output, quiet=False)
-	
-	#-----------------------------------
-	
-	model.load_weights(home+'/.deepface/weights/openface_weights.h5')
-	
-	#-----------------------------------
-	
-	return model
\ No newline at end of file
+    myInput = Input(shape=(96, 96, 3))
+
+    x = ZeroPadding2D(padding=(3, 3), input_shape=(96, 96, 3))(myInput)
+    x = Conv2D(64, (7, 7), strides=(2, 2), name='conv1')(x)
+    x = BatchNormalization(axis=3, epsilon=0.00001, name='bn1')(x)
+    x = Activation('relu')(x)
+    x = ZeroPadding2D(padding=(1, 1))(x)
+    x = MaxPooling2D(pool_size=3, strides=2)(x)
+    x = Lambda(lambda x: tf.nn.lrn(x, alpha=1e-4, beta=0.75), name='lrn_1')(x)
+    x = Conv2D(64, (1, 1), name='conv2')(x)
+    x = BatchNormalization(axis=3, epsilon=0.00001, name='bn2')(x)
+    x = Activation('relu')(x)
+    x = ZeroPadding2D(padding=(1, 1))(x)
+    x = Conv2D(192, (3, 3), name='conv3')(x)
+    x = BatchNormalization(axis=3, epsilon=0.00001, name='bn3')(x)
+    x = Activation('relu')(x)
+    x = Lambda(lambda x: tf.nn.lrn(x, alpha=1e-4, beta=0.75), name='lrn_2')(x)
+    x = ZeroPadding2D(padding=(1, 1))(x)
+    x = MaxPooling2D(pool_size=3, strides=2)(x)
+
+    # Inception3a
+    inception_3a_3x3 = Conv2D(96, (1, 1), name='inception_3a_3x3_conv1')(x)
+    inception_3a_3x3 = BatchNormalization(
+        axis=3,
+        epsilon=0.00001,
+        name='inception_3a_3x3_bn1'
+    )(inception_3a_3x3)
+    inception_3a_3x3 = Activation('relu')(inception_3a_3x3)
+    inception_3a_3x3 = ZeroPadding2D(padding=(1, 1))(inception_3a_3x3)
+    inception_3a_3x3 = Conv2D(
+        128, (3, 3),
+        name='inception_3a_3x3_conv2'
+    )(inception_3a_3x3)
+    inception_3a_3x3 = BatchNormalization(
+        axis=3,
+        epsilon=0.00001,
+        name='inception_3a_3x3_bn2'
+    )(inception_3a_3x3)
+    inception_3a_3x3 = Activation('relu')(inception_3a_3x3)
+
+    inception_3a_5x5 = Conv2D(16, (1, 1), name='inception_3a_5x5_conv1')(x)
+    inception_3a_5x5 = BatchNormalization(
+        axis=3,
+        epsilon=0.00001,
+        name='inception_3a_5x5_bn1'
+    )(inception_3a_5x5)
+    inception_3a_5x5 = Activation('relu')(inception_3a_5x5)
+    inception_3a_5x5 = ZeroPadding2D(padding=(2, 2))(inception_3a_5x5)
+    inception_3a_5x5 = Conv2D(
+        32, (5, 5),
+        name='inception_3a_5x5_conv2'
+    )(inception_3a_5x5)
+    inception_3a_5x5 = BatchNormalization(
+        axis=3,
+        epsilon=0.00001,
+        name='inception_3a_5x5_bn2'
+    )(inception_3a_5x5)
+    inception_3a_5x5 = Activation('relu')(inception_3a_5x5)
+
+    inception_3a_pool = MaxPooling2D(pool_size=3, strides=2)(x)
+    inception_3a_pool = Conv2D(
```

Comment:
om eerlijk te zijn vind ik deze files wel een goed voorbeeld waar de line width 79 niet goed werkt. dit is een stuk minder leesbaar imo. kan het overleven als je het zo laat overigens

Reply:
Ik had deze files sowieso anders ingeschoten qua formatting idd, maar voor nu heb ik het even zo gedaan om iig automatische formatting mogelijk te maken